docs/source/gateway/index.rst

```diff
@@ -891,19 +891,25 @@ This endpoint allows you to submit a query to a configured provider route. The d
 
 .. code-block:: bash
 
-    curl -X POST http://my.gateway:8888/gateway/completions/invocations -H "Content-Type: application/json" -d '{"prompt": "Describe the probability distribution of the decay chain of U-235"}'
+    curl -X POST http://my.gateway:8888/api/2.0/gateway/gateway/completions/invocations \
+      -H "Content-Type: application/json" \
+      -d '{"prompt": "Describe the probability distribution of the decay chain of U-235"}'
 
 * ``Chat``
 
 .. code-block:: bash
 
-    curl -X POST http://my.gateway:8888/gateway/chat/invocations -H "Content-Type: application/json" -d '{"messages": [{"role": "user", "content": "Can you write a limerick about orange flavored popsicles?"}]}'
+    curl -X POST http://my.gateway:8888/gateway/chat/invocations \
+      -H "Content-Type: application/json" \
+      -d '{"messages": [{"role": "user", "content": "Can you write a limerick about orange flavored popsicles?"}]}'
 
 * ``Embeddings``
 
 .. code-block:: bash
 
-    curl -X POST http://my.gateway:8888/gateway/embeddings/invocations -H "Content-Type: application/json" -d '{"texts": ["I'd like to return my shipment of beanie babies, please", "Can I please speak to a human now?"]}'
+    curl -X POST http://my.gateway:8888/api/2.0/gateway/gateway/embeddings/invocations \
```

Comment:
for invocations, we shouldn't have `/api/2.0`. We should only have `/api/2.0` for CRUD